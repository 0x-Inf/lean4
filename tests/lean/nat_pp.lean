#reduce nat.add (nat.of_num 3) (nat.of_num 6)
open nat
#reduce nat.add (nat.of_num 3) (nat.of_num 6)
#reduce (3:nat) + 6
