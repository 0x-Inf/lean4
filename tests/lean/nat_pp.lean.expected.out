nat.succ (nat.succ (nat.succ (nat.succ (nat.succ (nat.succ (nat.succ (nat.succ (nat.succ nat.zero))))))))
9
9
